docs/working_with_page.md

```diff
@@ -0,0 +1,226 @@
+    The block will display Content items of the selected Content Types that are in the selected parent folder.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Banner block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Image** section, click **Select content**, navigate through the content and select an image to display.
+    - In the **URL** box, enter a URL to open when clicking the selected image.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Collection block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Location list** section, click **Select Content**, navigate through the content 
+    and add to the collection Content items of any Content Type you want.
+    All selected Content items appear in the **Confirmed items** box at the bottom of the window.
+    When done selecting, click **Confirm**.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Gallery block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Folder** section, click **Select content**, navigate through the content, select a folder containing images to display and click **Confirm**. 
+    After submitting the settings, all images in the folder will appear in the Gallery block. 
+    Note that selecting a folder containing Content items other than images results in displaying only a link to the folder they are stored in.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Video block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Video** section, click **Select content**, navigate through the content, select a video to display in the block and click **Confirm**.
+    In the **Basic** tab you can preview the selected video before adding it to the Page.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### RSS block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **URL** box, provide the URL for the RSS news feed.
+    - In the **Limit** box, enter a number or use the up and down buttons to set the limit of news items appearing in the block.
+    - In the **Offset** box, enter a number or use the up and down buttons to set the limit of featured news items appearing in the block.
+    
+    ### Content Scheduler block
```

Comment:
This section duplicates publishing/advanced_publishing_options/ again. You only need a short description and a link to that page.